exchange/src/exchange/tickers/engine/new_engine.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include "order_container.hpp"
+#include "shared/util.hpp"
+
+using MarketOrder = nutc::messages::MarketOrder;
+using ObUpdate = nutc::messages::ObUpdate;
+using Match = nutc::messages::Match;
+using SIDE = nutc::messages::SIDE;
+
```

Comment:
you can't do this in header files

it propagates to all files that include the header file

have to do it inside a namespace at least

Reply:
its a carryover from engine itll be removed